robottelo/ui/contentviews.py

```diff
@@ -722,3 +722,40 @@ def fetch_puppet_module(self, cv_name, module_name):
             else:
                 raise UINoSuchElementError(
                     "Couldn't find puppet-modules tab")
+
+    def copy_view(self, name, new_name=None):
+        """
+        Copies an existing Content View
+        """
+        cv = self.search(name)
+        if cv and new_name:
+            cv.click()
+            self.wait_for_ajax()
+            self.edit_entity(locators['contentviews.copy'],
+                             locators['contentviews.copy_name'],
+                             new_name, locators['ak.copy_create'])
+        else:
+            raise UIError('Could not copy the Content View %s .', name)
+
+    def fetch_yum_content_repo_name(self, cv_name):
+        """
+        Fetch associated content host from selected activation key
+        """
+        # find content_view
+        cv = self.search(cv_name)
+        if not cv:
```

Comment:
Performing an explicit comparison against `None` may prevent some subtle bugs.

    if cv is None